src/WP_CLI/CommandWithTranslation.php

```diff
@@ -133,24 +133,27 @@ protected function sort_translations_callback( $a, $b ) {
 	 */
 	public function install( $args, $assoc_args ) {
 
-		list( $language_code ) = $args;
+		$language_codes = $args;
 
```

Comment:
Because only one language can be active at a time, we should return an error when `--activate` is used with more than one language?